avro_schema/il.lua

```diff
@@ -9,7 +9,7 @@ local insert, remove = table.insert, table.remove
 local concat         = table.concat
 local max            = math.max
 
-ffi.cdef([[
+pcall(ffi.cdef, [[
```

Comment:
I would do it strictier: check for ffi.C.schema_il_Opcode and if it present, then assert that ffi.C.schema_il_V is here too. If it is not present assert that ffi.C.schema_il_V is not present too and do ffi.cdef w/o pcall.

Reply:
It seems pcall(ffi.new, 'struct schema_il_Opcode') does the trick.

Reply:
Fixed

And test case was added